test/pj-rehearse-integration/expected.yaml

```diff
@@ -512,11 +512,6 @@
         entrypoint: gcr.io/k8s-prow/entrypoint:v20190129-0a3c54c
         initupload: gcr.io/k8s-prow/initupload:v20190129-0a3c54c
         sidecar: gcr.io/k8s-prow/sidecar:v20190129-0a3c54c
-    extra_refs:
-    - base_ref: cluster-profile
-      org: openshift
-      repo: origin
-      workdir: true
```

Comment:
This looks suspicious... This does not seem like the case where `extra_refs` should be removed (refs are not nil and are openshift/release, extra_refs are openshift/origin)

:confused:

Reply:
No, quite the opposite. This presubmit is under the `openshift/origin` repo in the [job config](https://github.com/openshift/ci-tools/blob/master/test/pj-rehearse-integration/master/ci-operator/jobs/super/duper/super-duper-cluster-profile-presubmits.yaml#L2).

Reply:
Well, yeah - that's why I would expect it to have openshift/release refs (to report stuff to github) and openshift/origin extra_refs with workdir: true to be able to work with its code.

Reply:
OH! yea